include/librealsense2/h/rs_option.h

```diff
@@ -117,6 +117,7 @@ extern "C" {
         RS2_OPTION_AUTO_GAIN_LIMIT_TOGGLE, /**< Enable / disable color image auto-gain*/
         RS2_OPTION_EMITTER_FREQUENCY, /**< Select emitter (laser projector) frequency, see rs2_emitter_frequency for values */
         RS2_OPTION_DEPTH_AUTO_EXPOSURE_MODE, /**< Select depth sensor auto exposure mode see rs2_depth_auto_exposure_mode for values  */
+        RS2_OPTION_LEFT_IR_TEMPERATURE, /**< Temperature of the Left IR Sensor */
```

Comment:
Not sure D450 OHM will have this temperature. 
@maloel this is something we need to verify.

Reply:
IMO it is registering it in this PR

Reply:
is there some open issue here?

Reply:
I noted this -- nothing to do here, Remi